lib/matplotlib/axes/_axes.py

```diff
@@ -2420,9 +2444,16 @@ def stem(self, *args, **kwargs):
         except IndexError:
             basefmt = kwargs.pop('basefmt', 'r-')
 
+        # Check the orientation variable to see if the user
+        # wants a vertical or horizontal stem plot
+        orientation = kwargs.pop('orientation', 'horizontal')
```

Comment:
should probably do some validation here to make sure 'aardvark' or the like does not get passed in.